### PR TITLE
Try to address Issue #188

### PR DESCRIPTION
**Pull request by NicsTr**
Try to address Issue #188

### CODE REVIEW

File: benchmarks/benchmark-echelon.C [hunk 1]
```diff
@@ -142,16 +145,18 @@ void verification_PLUQ(const Field & F, typename Field::Element * B, typename Fi
 	L = FFLAS::fflas_new<Field::Element>(m*R);
 	U = FFLAS::fflas_new<Field::Element>(R*n);
 	
-	PARFOR1D (i,m*R,H, F.init(L[i], 0.0); );
+	PARFOR1D (i,m*R,H, F.init(L[i]); F.assign(L[i], 0.0); );
```
Comment (Breush) [inline benchmarks/benchmark-echelon.C]: According to @jgdumas, we should have braces around the two instructions.

File: tests/test-igemm.C [hunk 2]
```diff
@@ -142,27 +142,33 @@ int test_igemm(size_t m, size_t n, size_t k, enum FFLAS_TRANSPOSE tA, enum FFLAS
...
 	for (size_t i=0;i<rb;++i)
 		for (size_t j=0;j<cb;++j)
-			F.init(Bi[i*ldB+j],B[i*ldb+j]);
+			F.init(Bi[i*ldB+j]);
+			F.assign(Bi[i*ldB+j],B[i*ldb+j]);
 	for (size_t i=0;i<m;++i)
 		for (size_t j=0;j<n;++j)
-			F.init(Ci[i*ldC+j],D[i*n+j]);
+			F.init(Ci[i*ldC+j]);
+			F.assign(Ci[i*ldC+j],D[i*n+j]);
```
Comment (Breush) [inline tests/test-igemm.C]: 9 braces pairs missing in this file

File: tests/test-ppluq.C [hunk 3]
```diff
@@ -90,23 +90,28 @@ void verification_PLUQ(const Field & F, typename Field::Element * B, typename Fi
   ParSeqHelper::Parallel H;
 
   PARFOR1D (i,m*R, H,
-    F.init(L[i], 0.0);
+    F.init(L[i]);
+    F.assign(L[i], 0.0);
```
Comment (Breush) [inline tests/test-ppluq.C]: Braces for each of this file

File: fflas-ffpack/fflas/fflas_fgemv.inl [hunk 4]
```diff
@@ -54,9 +54,11 @@ namespace FFLAS{ namespace Protected {
 		Givaro::ModularBalanced<FloatElement> G((FloatElement) F.characteristic());
 		FloatElement tmp,alphaf, betaf;
 		F.convert (tmp, beta);
-		G.init(betaf,tmp);
+		G.init(betaf);
```
Comment (ClementPernet) [inline fflas-ffpack/fflas/fflas_fgemv.inl]: keep it as before: 1 init with 2 args

File: fflas-ffpack/fflas/fflas_fgemv.inl [hunk 5]
```diff
@@ -54,9 +54,11 @@ namespace FFLAS{ namespace Protected {
...
+		G.assign(betaf,tmp);
 		F.convert (tmp, alpha);
-		G.init(alphaf,tmp);
+		G.init(alphaf);
```
Comment (ClementPernet) [inline fflas-ffpack/fflas/fflas_fgemv.inl]: keep it as before: 1 init with 2 args

File: fflas-ffpack/fflas/fflas_freduce.h [hunk 6]
```diff
@@ -98,11 +98,15 @@ namespace FFLAS {
 		ConstOtherElement_ptr Yi = Y ;
 
 		if (incX == 1 && incY == 1)
-			for (; Yi < Y + n ; ++Xi, ++Yi)
-				F.init( *Xi , *Yi);
+			for (; Yi < Y + n ; ++Xi, ++Yi) {
```
Comment (ClementPernet) [inline fflas-ffpack/fflas/fflas_freduce.h]: keep it as before: 1 init with 2 args

File: fflas-ffpack/fflas/fflas_freduce.h [hunk 7]
```diff
@@ -98,11 +98,15 @@ namespace FFLAS {
...
+				F.init( *Xi );
+				F.assign( *Xi , *Yi);
+			}
 		else
-			for (; Yi < Y+n*incY; Xi+=incX, Yi += incY )
-				F.init( *Xi , *Yi);
+			for (; Yi < Y+n*incY; Xi+=incX, Yi += incY ) {
```
Comment (ClementPernet) [inline fflas-ffpack/fflas/fflas_freduce.h]: keep it as before: 1 init with 2 args

File: fflas-ffpack/fflas/fflas_fsyr2k.inl [hunk 8]
```diff
@@ -51,7 +51,8 @@ namespace FFLAS {
             size_t incA = (trans==FFLAS::FflasNoTrans)?1:lda;
             size_t incB = (trans==FFLAS::FflasNoTrans)?1:ldb;
             typename Field::Element two;
-            F.init(two, 2);
+            F.init(two);
```
Comment (ClementPernet) [inline fflas-ffpack/fflas/fflas_fsyr2k.inl]: keep it as before: 1 init with 2 args

File: fflas-ffpack/fflas/fflas_sparse/read_sparse.h [hunk 9]
```diff
@@ -148,7 +148,8 @@ namespace FFLAS {
 				typename Field::Element v;
 
 				assert(l && c);
-				f.init(v, d);
+				f.init(v);
```
Comment (ClementPernet) [inline fflas-ffpack/fflas/fflas_sparse/read_sparse.h]: keep it as before: 1 init with 2 args

File: fflas-ffpack/fflas/fflas_sparse/read_sparse.h [hunk 10]
```diff
@@ -248,7 +249,8 @@ namespace FFLAS {
 				index_t c = std::stoull(tokens[2 * i + 1]) - 1;
 				typename Field::Element v;
 				int64_t vtmp = std::stoll(tokens[2 * (i + 1)]);
-				f.init(v, vtmp);
+				f.init(v);
```
Comment (ClementPernet) [inline fflas-ffpack/fflas/fflas_sparse/read_sparse.h]: keep it as before: 1 init with 2 args

File: fflas-ffpack/fflas/fflas_sparse/read_sparse.h [hunk 11]
```diff
@@ -323,14 +325,16 @@ namespace FFLAS {
 		if (field_desc ==1) { /*  modulo */
 			T modulo_read ;
 			file.read((char*) &modulo_read, sizeof(T));
-			F.init(modulo,modulo_read);
+			F.init(modulo);
```
Comment (ClementPernet) [inline fflas-ffpack/fflas/fflas_sparse/read_sparse.h]: keep it as before: 1 init with 2 args

File: fflas-ffpack/fflas/fflas_sparse/read_sparse.h [hunk 12]
```diff
@@ -323,14 +325,16 @@ namespace FFLAS {
...
+			F.assign(modulo,modulo_read);
 		}
 		/*  do something with field_desc and multiprec... */
 		T * data_read = fflas_new<T>(dims);
 		file.read((char*)data_read,sizeof(T));
 		/* TODO freduce ? */
 		for (size_t i = 0 ; i< dims ; ++i) {
-			F.init(val[i],data_read[i]);
+			F.init(val[i]);
```
Comment (ClementPernet) [inline fflas-ffpack/fflas/fflas_sparse/read_sparse.h]: keep it as before: 1 init with 2 args

File: fflas-ffpack/ffpack/ffpack_fsytrf.inl [hunk 13]
```diff
@@ -141,7 +141,8 @@ namespace FFPACK {
 	
 		size_t rank = 0;
         typename Field::Element two;
-        F.init(two, INT32_C(2));
+        F.init(two);
```
Comment (ClementPernet) [inline fflas-ffpack/ffpack/ffpack_fsytrf.inl]: keep it as before: 1 init with 2 args

File: fflas-ffpack/ffpack/ffpack_ftrssyr2k.inl [hunk 14]
```diff
@@ -39,7 +39,8 @@ namespace FFPACK {
         if (!N) return;
         if (N <= 1){ // base case TODO: write a basecase with dimension = threshold
             typename Field::Element two;
-            F.init(two,2);
+            F.init(two);
```
Comment (ClementPernet) [inline fflas-ffpack/ffpack/ffpack_ftrssyr2k.inl]: keep it as before: 1 init with 2 args

File: fflas-ffpack/field/rns-double.h [hunk 15]
```diff
@@ -334,7 +334,8 @@ namespace FFPACK {
 				_negbasis[i] = 0-_basis[i];
 				_field_rns[i] = ModField(_basis[i]);
 				_Mi[i]        = _M/(uint64_t)_basis[i];
-				_field_rns[i].init(_MMi[i], _Mi[i] % (double)_basis[i]);
+				_field_rns[i].init(_MMi[i]);
```
Comment (ClementPernet) [inline fflas-ffpack/field/rns-double.h]: keep it as before: 1 init with 2 args

File: fflas-ffpack/utils/fflas_randommatrix.h [hunk 16]
```diff
@@ -434,8 +434,10 @@ namespace FFPACK{
         RandomTriangularMatrix (F, N, N, FFLAS::FflasUpper, FFLAS::FflasNonUnit, true, U, N, G);
 
         typename Field::Element alpha, beta;
-        F.init(alpha,1.0);
-        F.init(beta,0.0);
+        F.init(alpha);
```
Comment (ClementPernet) [inline fflas-ffpack/utils/fflas_randommatrix.h]: Replace alpha,beta by F.one and F.zero in the call to fgemm below and remove them

File: tests/regression-check.C [hunk 17]
```diff
@@ -110,8 +110,10 @@ bool gf2ModularBalanced(){
 	Field::Element_ptr G, H;
 	H = fflas_new(F,3,7);
 
-	for (unsigned i=0; i<21; i++)
-        F.init(H[i], h_[i]);
+	for (unsigned i=0; i<21; i++) {
+        F.init(H[i]);
```
Comment (ClementPernet) [inline tests/regression-check.C]: keep it as before: 1 init with 2 args

File: tests/test-fgesv.C [hunk 18]
```diff
@@ -64,8 +64,10 @@ int main(int argc, char** argv){
 	}
 	int nbit=atoi(argv[4]); // number of times the product is performed
 	Field F(atoi(argv[1]));
-	F.init(zero,0.0);
-	F.init(one,1.0);
+	F.init(zero);
```
Comment (ClementPernet) [inline tests/test-fgesv.C]: replace these one and zero by F.one and F.zero where they are used (fgemm below) and remove them.

File: tests/test-frobenius.C [hunk 19]
```diff
@@ -70,7 +70,8 @@ int main(int argc, char** argv){
 	}
 	Field F( atoi(argv[1]) );
 	Field::Element one;
-	F.init(one, 1U);
+	F.init(one);
```
Comment (ClementPernet) [inline tests/test-frobenius.C]: replace this one by F.one where it's used and remove it

File: tests/test-igemm.C [hunk 20]
```diff
@@ -231,23 +249,34 @@ int test_igemm(size_t m, size_t n, size_t k, enum FFLAS_TRANSPOSE tA, enum FFLAS
 		// Givaro::Modular<double> G(65537);
 		Givaro::ZRing<double> G;
 		double af, bf ;
-		G.init(af,alpha);
-		G.init(bf,beta);
+		G.init(af);
```
Comment (ClementPernet) [inline tests/test-igemm.C]: keep it as before: 1 init with 2 args

File: tests/test-igemm.C [hunk 21]
```diff
@@ -231,23 +249,34 @@ int test_igemm(size_t m, size_t n, size_t k, enum FFLAS_TRANSPOSE tA, enum FFLAS
...
+		G.assign(af,alpha);
+		G.init(bf);
```
Comment (ClementPernet) [inline tests/test-igemm.C]: keep it as before: 1 init with 2 args

File: tests/test-igemm.C [hunk 22]
```diff
@@ -140,29 +140,47 @@ int test_igemm(size_t m, size_t n, size_t k, enum FFLAS_TRANSPOSE tA, enum FFLAS
...
-			F.init(Ai[j*ldA+i],A[i*lda+j]);
-	for (size_t i=0;i<k;++i)
-		for (size_t j=0;j<n;++j)
-			F.init(Bi[j*ldB+i],B[i*ldb+j]);
-	for (size_t i=0;i<m;++i)
-		for (size_t j=0;j<n;++j)
-			F.init(Ci[j*ldC+i],D[i*n+j]);
+	for (size_t i=0;i<m;++i) {
+		for (size_t j=0;j<k;++j) {
+			F.init(Ai[j*ldA+i]);
```
Comment (ClementPernet) [inline tests/test-igemm.C]: keep it as before: 1 init with 2 args

File: tests/test-igemm.C [hunk 23]
```diff
@@ -140,29 +140,47 @@ int test_igemm(size_t m, size_t n, size_t k, enum FFLAS_TRANSPOSE tA, enum FFLAS
...
+			F.assign(Ai[j*ldA+i],A[i*lda+j]);
+        }
+    }
+	for (size_t i=0;i<k;++i) {
+		for (size_t j=0;j<n;++j) {
+			F.init(Bi[j*ldB+i]);
```
Comment (ClementPernet) [inline tests/test-igemm.C]: keep it as before: 1 init with 2 args

File: tests/test-igemm.C [hunk 24]
```diff
@@ -140,29 +140,47 @@ int test_igemm(size_t m, size_t n, size_t k, enum FFLAS_TRANSPOSE tA, enum FFLAS
...
+			F.assign(Bi[j*ldB+i],B[i*ldb+j]);
+        }
+    }
+	for (size_t i=0;i<m;++i) {
+		for (size_t j=0;j<n;++j) {
+			F.init(Ci[j*ldC+i]);
```
Comment (ClementPernet) [inline tests/test-igemm.C]: keep it as before: 1 init with 2 args

File: tests/test-igemm.C [hunk 25]
```diff
@@ -140,29 +140,47 @@ int test_igemm(size_t m, size_t n, size_t k, enum FFLAS_TRANSPOSE tA, enum FFLAS
...
-			F.init(Ai[i*ldA+j],A[i*lda+j]);
-	for (size_t i=0;i<rb;++i)
-		for (size_t j=0;j<cb;++j)
-			F.init(Bi[i*ldB+j],B[i*ldb+j]);
-	for (size_t i=0;i<m;++i)
-		for (size_t j=0;j<n;++j)
-			F.init(Ci[i*ldC+j],D[i*n+j]);
+	for (size_t i=0;i<ra;++i) {
+		for (size_t j=0;j<ca;++j) {
+			F.init(Ai[i*ldA+j]);
```
Comment (ClementPernet) [inline tests/test-igemm.C]: keep it as before: 1 init with 2 args

File: tests/test-igemm.C [hunk 26]
```diff
@@ -140,29 +140,47 @@ int test_igemm(size_t m, size_t n, size_t k, enum FFLAS_TRANSPOSE tA, enum FFLAS
...
+			F.assign(Ai[i*ldA+j],A[i*lda+j]);
+        }
+    }
+	for (size_t i=0;i<rb;++i) {
+		for (size_t j=0;j<cb;++j) {
+			F.init(Bi[i*ldB+j]);
```
Comment (ClementPernet) [inline tests/test-igemm.C]: keep it as before: 1 init with 2 args

File: tests/test-igemm.C [hunk 27]
```diff
@@ -140,29 +140,47 @@ int test_igemm(size_t m, size_t n, size_t k, enum FFLAS_TRANSPOSE tA, enum FFLAS
...
+			F.assign(Bi[i*ldB+j],B[i*ldb+j]);
+        }
+    }
+	for (size_t i=0;i<m;++i) {
+		for (size_t j=0;j<n;++j) {
+			F.init(Ci[i*ldC+j]);
```
Comment (ClementPernet) [inline tests/test-igemm.C]: keep it as before: 1 init with 2 args

File: tests/test-igemm.C [hunk 28]
```diff
@@ -231,23 +249,34 @@ int test_igemm(size_t m, size_t n, size_t k, enum FFLAS_TRANSPOSE tA, enum FFLAS
...
-				G.init(Af[i*ldA+j],A[i*lda+j]);
-		for (size_t i=0;i<rb;++i)
-			for (size_t j=0;j<cb;++j)
-				G.init(Bf[i*ldB+j],B[i*ldb+j]);
-		for (size_t i=0;i<m;++i)
-			for (size_t j=0;j<n;++j)
-				G.init(Cf[i*ldC+j],D[i*n+j]);
+		for (size_t i=0;i<ra;++i) {
+			for (size_t j=0;j<ca;++j) {
+				G.init(Af[i*ldA+j]);
```
Comment (ClementPernet) [inline tests/test-igemm.C]: keep it as before: 1 init with 2 args

File: tests/test-igemm.C [hunk 29]
```diff
@@ -231,23 +249,34 @@ int test_igemm(size_t m, size_t n, size_t k, enum FFLAS_TRANSPOSE tA, enum FFLAS
...
+				G.assign(Af[i*ldA+j],A[i*lda+j]);
+            }
+        }
+		for (size_t i=0;i<rb;++i) {
+			for (size_t j=0;j<cb;++j) {
+				G.init(Bf[i*ldB+j]);
```
Comment (ClementPernet) [inline tests/test-igemm.C]: keep it as before: 1 init with 2 args

File: tests/test-igemm.C [hunk 30]
```diff
@@ -231,23 +249,34 @@ int test_igemm(size_t m, size_t n, size_t k, enum FFLAS_TRANSPOSE tA, enum FFLAS
...
+				G.assign(Bf[i*ldB+j],B[i*ldb+j]);
+            }
+        }
+		for (size_t i=0;i<m;++i) {
+			for (size_t j=0;j<n;++j) {
+				G.init(Cf[i*ldC+j]);
```
Comment (ClementPernet) [inline tests/test-igemm.C]: keep it as before: 1 init with 2 args

File: tests/test-krylov-elim.C [hunk 31]
```diff
@@ -68,7 +68,8 @@ int main(int argc, char** argv){
 	}
 	Field F(atoi(argv[1]));
 	Field::Element one;
-	F.init(one, 1U);
+	F.init(one);
```
Comment (ClementPernet) [inline tests/test-krylov-elim.C]: replace one by F.one and remove it

File: tests/test-lu.C [hunk 32]
```diff
@@ -170,8 +170,10 @@ bool verifPLUQ (const Field & F, typename Field::ConstElement_ptr A, size_t lda,
 	fzero(F, R, n, U, n);
 	
 	typename Field::Element zero,one;
-	F.init(zero,0.0);
-	F.init(one,1.0);
+	F.init(zero);
```
Comment (ClementPernet) [inline tests/test-lu.C]: replace one,zero, by F.one, F.zero and remove them

File: tests/test-permutations.C [hunk 33]
```diff
@@ -50,14 +50,21 @@ bool checkMonotonicApplyP(FFLAS_SIDE Side, FFLAS_TRANSPOSE trans, size_t * P, si
...
-			for (size_t j = 0; j<M; ++j)
-				F.init(A[i*lda+j],i*10+j);
-	else
-		for (size_t i = 0; i<N; ++i)
-			for (size_t j = 0; j<M; ++j)
-				F.init(A[i+j*lda],i*10+j);
+	if (Side == FflasLeft) {
+		for (size_t i = 0; i<N; ++i) {
+			for (size_t j = 0; j<M; ++j) {
+				F.init(A[i*lda+j]);
```
Comment (ClementPernet) [inline tests/test-permutations.C]: keep it as before: 1 init with 2 args

File: tests/test-permutations.C [hunk 34]
```diff
@@ -50,14 +50,21 @@ bool checkMonotonicApplyP(FFLAS_SIDE Side, FFLAS_TRANSPOSE trans, size_t * P, si
...
+				F.assign(A[i*lda+j],i*10+j);
+			}
+		}
+	} else {
+		for (size_t i = 0; i<N; ++i) {
+			for (size_t j = 0; j<M; ++j) {
+				F.init(A[i+j*lda]);
```
Comment (ClementPernet) [inline tests/test-permutations.C]: keep it as before: 1 init with 2 args

File: tests/test-pfgemm-DSL.C [hunk 35]
```diff
@@ -155,7 +157,8 @@ BEGIN_PARALLEL_MAIN(int argc, char** argv)
 #if __FFLASFFPACK_DEBUG
         bool wrong = false;
 	Field::Element zero;
-        F.init(zero, 0.0);
+        F.init(zero);
```
Comment (ClementPernet) [inline tests/test-pfgemm-DSL.C]: replace zero by F.zero and remove it

File: tests/test-pluq.C [hunk 36]
```diff
@@ -168,8 +168,10 @@ int main(int argc, char** argv){
 	U = FFLAS::fflas_new<Field::Element>(R*n);
 
 	Field::Element zero,one;
-	F.init(zero,0.0);
-	F.init(one,1.0);
+	F.init(zero);
```
Comment (ClementPernet) [inline tests/test-pluq.C]: replace zero,one by F.zero and F.one and remove them

File: tests/test-ppluq.C [hunk 37]
```diff
@@ -89,24 +89,16 @@ void verification_PLUQ(const Field & F, typename Field::Element * B, typename Fi
...
-
-  PARFOR1D (i,m*R, H,
-    F.init(U[i], 0.0);
-            );
-
-
-  PARFOR1D (i,m*n, H,
-    F.init(X[i], 0.0);
-            );
+  PARFOR1D (i,m*R, H, { F.init(L[i]); F.assign(L[i], 0.0); } );
```
Comment (ClementPernet) [inline tests/test-ppluq.C]: replace 0.0 by F.zero

File: tests/test-ppluq.C [hunk 38]
```diff
@@ -89,24 +89,16 @@ void verification_PLUQ(const Field & F, typename Field::Element * B, typename Fi
...
+  PARFOR1D (i,m*R, H, { F.init(U[i]); F.assign(U[i], 0.0); } );
+  PARFOR1D (i,m*n, H, { F.init(X[i]); F.assign(X[i], 0.0); } );
 
 
   Field::Element zero,one;
-  F.init(zero,0.0);
-  F.init(one,1.0);
+  F.init(zero);
```
Comment (ClementPernet) [inline tests/test-ppluq.C]: replace zero,one by F.zero,F.one and remove them

File: tests/test-ppluq.C [hunk 39]
```diff
@@ -190,8 +182,10 @@ int main(int argc, char** argv)
 	// Field::RandIter G(F, seed1);
 
 	Field::Element alpha, beta;
-	F.init(alpha,1.0);
-	F.init(beta,0.0);
+	F.init(alpha);
+	F.assign(alpha,1.0);
```
Comment (ClementPernet) [inline tests/test-ppluq.C]: replace zero, one by F.zero and F.one

File: tests/testeur_ftrsm.C [hunk 40]
```diff
@@ -84,8 +84,10 @@ int main(int argc, char** argv){
 		}while( (p <= 2) );
 
 		Field F (p);
-		F.init (zero,0.0);
-		F.init (one,1.0);
+		F.init(zero);
```
Comment (ClementPernet) [inline tests/testeur_ftrsm.C]: replace zero,one by F.zero and F.one and remove them

File: tests/testeur_lqup.C [hunk 41]
```diff
@@ -88,8 +88,10 @@ int main(int argc, char** argv){
 		}while( (p <= 2) );
 
 		Field F( p);
-		F.init(zero,0.0);
-		F.init(one,1.0);
+		F.init(zero);
```
Comment (ClementPernet) [inline tests/testeur_lqup.C]: replace zero,one by F.zero F.one and remove them

File: tutorials/101-fgemm.C [hunk 42]
```diff
@@ -43,11 +43,16 @@ int main(int argc, char** argv) {
 	C = fflas_new(F,2,2);  
 
 	F.assign(*(A+0),F.one);
-	F.init(*(A+1),2);
-	F.init(*(A+2),3);
-	F.init(*(A+3),5);
-	F.init(*(A+4),7);
-	F.init(*(A+5),11);
+	F.init(*(A+1));
```
Comment (ClementPernet) [inline tutorials/101-fgemm.C]: keep it as before: 1 init with 2 args

File: tutorials/fflas_101.C [hunk 43]
```diff
@@ -106,8 +106,10 @@ int main(int argc, char** argv) {
   // already exixting elements.
   // The first way is using init().
   
-  F1.init(a,2);
-  F1.init(b,205);
+  F1.init(a);
```
Comment (ClementPernet) [inline tutorials/fflas_101.C]: keep it as before: 1 init with 2 args

File: tutorials/fflas_101.C [hunk 44]
```diff
@@ -160,7 +162,8 @@ int main(int argc, char** argv) {
   std::cout << "That's because -3 is not"<< std::endl;
   std::cout << "an element of the field."<< std::endl;
   std::cout << "Thus -3 ought to be converted first."<< std::endl;
-  F1.init(h,-3);
+  F1.init(h);
```
Comment (ClementPernet) [inline tutorials/fflas_101.C]: keep it as before: 1 init with 2 args

File: tutorials/fflas_101_lvl1.C [hunk 45]
```diff
@@ -145,7 +145,8 @@ int main(int argc, char** argv) {
 
   // faxpby is a more general form of faxpy
 
-  //F.init(beta,14);
+  //F.init(beta);
```
Comment (ClementPernet) [inline tutorials/fflas_101_lvl1.C]: keep it as before: 1 init with 2 args

File: tutorials/ffpack-fgesv.C [hunk 46]
```diff
@@ -81,14 +81,22 @@ int main(int argc, char** argv) {
...
   // Fulfill the matrix b with desired values
-  F.init(b[0],4);
-  F.init(b[1],4);
-  F.init(b[2],6);
-  F.init(b[3],3);
-  F.init(b[4],3);
-  F.init(b[5],6);
-  F.init(b[6],4);
-  F.init(b[7],4);
+  F.init(b[0]);
```
Comment (ClementPernet) [inline tutorials/ffpack-fgesv.C]: keep it as before: 1 init with 2 args

File: tutorials/ffpack-solve.C [hunk 47]
```diff
@@ -54,15 +54,18 @@ int main(int argc, char** argv) {
   F.assign(A[1],F.zero);
   F.assign(A[2],F.zero);
   F.assign(A[3],F.zero);
-  F.init(A[4],15);
+  F.init(A[4]);
```
Comment (ClementPernet) [inline tutorials/ffpack-solve.C]: keep it as before: 1 init with 2 args

File: tutorials/ffpack-solve.C [hunk 48]
```diff
@@ -54,15 +54,18 @@ int main(int argc, char** argv) {
...
+  F.assign(A[4],15);
   F.assign(A[5],F.one);  
   F.assign(A[6],F.zero);
   F.init(A[7],3);  
   F.assign(A[8],F.zero);
   F.assign(A[9],F.zero);
   F.assign(A[10],F.one);
-  F.init(A[11],2);
-  F.init(A[12],10);
+  F.init(A[11]);
```
Comment (ClementPernet) [inline tutorials/ffpack-solve.C]: keep it as before: 1 init with 2 args

File: tutorials/ffpack-solve.C [hunk 49]
```diff
@@ -85,10 +88,14 @@ int main(int argc, char** argv) {
   b = fflas_new(F,M,1);
 
   // Fulfill the vector b with desired values
-  F.init(b[0],1);
-  F.init(b[1],3);
-  F.init(b[2],6);
-  F.init(b[3],5);
+  F.init(b[0]);
```
Comment (ClementPernet) [inline tutorials/ffpack-solve.C]: keep it as before: 1 init with 2 args

**Comment by NicsTr** [comment]
It should be ready to be merged now.

However, there can be some `init` left that were not caught by the original regexp.

**Comment by NicsTr** [comment]
Now everything should be correctly resolved.

**Comment by ClementPernet** [comment]
Great! Thanks.

**Pull request merged**